history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -951,15 +1045,29 @@ not only in word, but in deed also,
 
 I write this at the request of certain of my friends called Quakers.
 
-W+++.+++ SEWEL
+[.signed-section-signature]
+W+++.+++ Sewel
 
+[.signed-section-context-close]
 Amsterdam, the 23rd day of the month called April, 1679.
 
-Some questions presented to the Catholics, and the Pope of Rome,
-as the supreme head of their church, and commended to their consideration by George Fox.
+--
+
+[.embedded-content-document.treatise]
+--
+
+[.small-break]
```

Comment:
Looking at this in the PR, I may have put this in the wrong place...should have been above the embedded part?? I can't tell for sure.

Reply:
I think probably we can just remove it.  or does it look really funny without it?

Reply:
I deleted it, it looked fine without it. It was probably just a small break in the original, but isn't needed in this.